### ISSUE TITLE
Support for BTTV emote modifiers

### ISSUE THREAD

**Issue by Jnosh** [tommyxchow/frosty#269]
It would be great if Frosty supported the BTTV emote modifiers that were added in BTTV 7.5.0. Quoting from the release notes:

    Add "w! ", "h! ", "v! ", or "z! " before emote codes to render the following BetterTTV emote wide, flipped horizontally, flipped vertically, or zero-width space.

From what I understand these are implemented client side so Frosty would need to reimplement the same functionality.

**Comment by BigMood42069** [comment]
I agree with this, I would also love to see the emote modifiers added in. I'm commenting since there's not really a way to "upvote" an issue
